### ISSUE TITLE
LV_PRId32 and LV_PRIu32 compile error in lv_demo_benchmark

### ISSUE THREAD

**Issue by baldhead69** [kisvegabor/lvgl#3930]
Hi,
 
I want to run benchmark test from "https://github.com/lvgl/lv_demos/tree/master/src/lv_demo_benchmark" link, but compiler errors appear related with LV_PRId32 and LV_PRIu32.

```c
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c: In function 'scene_next_task_cb':
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:695:41: error: expected ')' before 'LV_PRId32'
         lv_label_set_text_fmt(title, "%"LV_PRId32"/%d: %s%s", scene_act * 2 + (opa_mode ? 1 : 0), (sizeof(scenes) / sizeof(scene_dsc_t) * 2) - 2,  scenes[scene_act].name, opa_mode ? " + opa" : "");
                                         ^~~~~~~~~
                                         )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:697:66: error: expected ')' before 'LV_PRId32'
             lv_label_set_text_fmt(subtitle, "Result of \"%s\": %"LV_PRId32" FPS", scenes[scene_act].name, scenes[scene_act].fps_normal);
                                                                  ^~~~~~~~~
                                                                  )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:700:76: error: expected ')' before 'LV_PRId32'
                 lv_label_set_text_fmt(subtitle, "Result of \"%s + opa\": %"LV_PRId32" FPS", scenes[scene_act - 1].name, scenes[scene_act - 1].fps_opa);
                                                                            ^~~~~~~~~
                                                                            )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:747:55: error: expected ')' before 'LV_PRIu32'
         lv_label_set_text_fmt(title, "Weighted FPS: %"LV_PRIu32, fps_weighted);
                                                       ^~~~~~~~~
                                                       )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:750:56: error: expected ')' before 'LV_PRIu32'
         lv_label_set_text_fmt(subtitle, "Opa. speed: %"LV_PRIu32"%%", opa_speed_pct);
                                                        ^~~~~~~~~
                                                        )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:791:50: error: expected ')' before 'LV_PRIu32'
                 lv_snprintf(buf, sizeof(buf), "%"LV_PRIu32, scenes[i].fps_normal);
                                                  ^~~~~~~~~
                                                  )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:804:50: error: expected ')' before 'LV_PRIu32'
                 lv_snprintf(buf, sizeof(buf), "%"LV_PRIu32, scenes[i].fps_opa);
                                                  ^~~~~~~~~
                                                  )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:829:46: error: expected ')' before 'LV_PRIu32'
             lv_snprintf(buf, sizeof(buf), "%"LV_PRIu32, scenes[i].fps_normal);
                                              ^~~~~~~~~
                                              )
../src/my_Files/graphic_Interface/lvgl/lv_demos-master/src/lv_demo_benchmark/lv_demo_benchmark.c:846:46: error: expected ')' before 'LV_PRIu32'
             lv_snprintf(buf, sizeof(buf), "%"LV_PRIu32, scenes[i].fps_opa);
                                              ^~~~~~~~~
```

**Comment by baldhead69** [comment]
Some suggestion.

I am using lvgl version 8.0.0 i think dev.

Thank's.

**Comment by kisvegabor** [comment]
Can you update to `release/v8.3`? I believe these are already fixed there.

**Comment by baldhead69** [comment]
I updated to release/v8.3 and worked.

First i tried to update to v9.0.0 dev, but problems with makefile appear.

https://www.microchip.com/forums/m/tm.aspx?m=1224283&fp=1

**Comment by baldhead69** [comment]
@kisvegabor 

I think that i found the problem:

```c
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg001.o:(.rodata.animimg001+0x0): multiple definition of `animimg001'; build/default/production/_ext/1010940615/animimg001.o:(.rodata.animimg001+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg001.o:(.rodata.animimg001_map+0x0): multiple definition of `animimg001_map'; build/default/production/_ext/1010940615/animimg001.o:(.rodata.animimg001_map+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg002.o:(.rodata.animimg002+0x0): multiple definition of `animimg002'; build/default/production/_ext/1010940615/animimg002.o:(.rodata.animimg002+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg002.o:(.rodata.animimg002_map+0x0): multiple definition of `animimg002_map'; build/default/production/_ext/1010940615/animimg002.o:(.rodata.animimg002_map+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg003.o:(.rodata.animimg003+0x0): multiple definition of `animimg003'; build/default/production/_ext/1010940615/animimg003.o:(.rodata.animimg003+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/animimg003.o:(.rodata.animimg003_map+0x0): multiple definition of `animimg003_map'; build/default/production/_ext/1010940615/animimg003.o:(.rodata.animimg003_map+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/img_caret_down.o:(.rodata.img_caret_down+0x0): multiple definition of `img_caret_down'; build/default/production/_ext/1010940615/img_caret_down.o:(.rodata.img_caret_down+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: build/default/production/_ext/443351157/img_caret_down.o:(.rodata.img_caret_down_map+0x0): multiple definition of `img_caret_down_map'; build/default/production/_ext/1010940615/img_caret_down.o:(.rodata.img_caret_down_map+0x0): first defined here
c:\microchip\xc32\v4.21\bin\bin\gcc\pic32mx\8.3.1\..\..\..\..\bin/pic32m-ld.exe: Link terminated due to previous error(s).
```

These files are with same name in two different folders:

```c
C:\Users\Rangel\Desktop\Mplab-Progs\pic32mz-teste\firmware\src\my_Files\graphic_Interface\lvgl\tests\src\test_assets\animimg001.c
C:\Users\Rangel\Desktop\Mplab-Progs\pic32mz-teste\firmware\src\my_Files\graphic_Interface\lvgl\examples\assets\animimg001.c
```

The same is true for other files:
* animimg001
* animimg002
* animimg003
* img_caret_down

**Comment by kisvegabor** [comment]
I've pushed a fix [ad8eed9](https://github.com/lvgl/lvgl/commit/ad8eed9eb58f6d6cc50720447f05921f1bc738d0).
Could you try it out?
